### DISCUSSION TITLE
Silicon Labs MG26 (EFR32MG26) SoC/MCU based adapters will offers up to 3200KB Flash and 512KB RAM in order to be as future-proof as possible

### DISCUSSION THREAD

**Discussion by Hedda** [darkxst/silabs-firmware-builder#81] [General]
FYI; Silicon Labs have now announced xG26 (EFR32xG26) family which includes multiprotocol MG26 SoC/MCU that Silabs call "future-proof":

https://news.silabs.com/2024-04-09-Silicon-Labs-xG26-Sets-New-Standard-in-Multiprotocol-Wireless-Device-Performance

https://www.silabs.com/wireless/efr32xg26

Silabs claim these are "future-proof" because feature up to a whopping 3200KB of Flash Storage  and 512KB of RAM, as well as 64 GPIO.

It looks like there already is platform support for 'efr32xg26'  in Silicon Labs Gecko SDK (GSDK) 4.4 / 4.4.x versions and later(?).

Now for Zigbee, Thread/OpenThread, and Bluetooth the interesting SoC/MCU for is Silicon Labs EFR32MG26 Series 2 (a.k.a. MG26):

https://www.silabs.com/wireless/zigbee/efr32mg26-series-2-socs

While there are no EFR32MG26/MGM260 based radio modules or adapers and dongles available yet I am sure that it will not take too long before we will at least see the first EFR32MG26 based radio module from companies like RF-Star/RFstar and Ebyte/ceByte.

Once they are made available I think this new EFR32MG26 SoC/MCU model will be particularly interesting as a Zigbee Coordinator targeting gateway applications that allow for a much higher total Zigbee 3.0 device limitation, (meaning there will be no problems using over 200 Zigbee 3.0 devices which have been a practical limitation on the popular EFR32MG21 with current firmware builds), while also providing enough headroom to enetually support RCP Multi-PAN running concurrent multiprotocol with multiple networks (like Zigbee and Matter over Thread Thread Boarder Router at the same time) if that feature ever becomes more stable in the future Gecko SDK versions.

https://www.silabs.com/wireless/zigbee

Other than having 5x Flash Storage and RAM + double of GPIOs, the specification looks practically identical to the previous EFR32MG24:

![image](https://github.com/darkxst/silabs-firmware-builder/assets/6320001/849d82e8-55ea-4160-8a70-c9d24a387a6b)

No news on FCC/CE precertified MGM260/MGM260P/MGM260L) modules based on EFR32MG26 to replace MGM240 and MGM240.

https://www.silabs.com/wireless/zigbee/efr32mg26-series-2-modules

https://www.silabs.com/wireless/zigbee/efr32mg24-series-2-modules

https://www.silabs.com/wireless/zigbee/efr32mg21-series-2-modules

PS: At the same time they also announced EFR32BG26 (Silabs EFR32 BG26) which is basically the same chip locked to Bluetooth only:

https://www.silabs.com/wireless/bluetooth/efr32bg26-series-2-socs

**Comment by darkxst** [comment]
I get the impression these were mostly aimed at EDGE AI applications, they otherwise identical to MG24 with the same core (however only a couple of mg24 models have the AI engine, presumably the mg26 will default with AI but not seen any data on this yet). Probably they wont provide much benefit for coordinators, but perhaps the extra resources may allow for otbr + ncp/rcp firmware on a single chip (esp32-c6 can do this already).

The "future proof" claims seem odd given series 3 chips are on the way.

**Reply by Hedda** [reply]
> The "future proof" claims seem odd given series 3 chips are on the way.

Their strategy with Series 3 is not clear to me at all, however I thought that the upcoming Silicon Labs Series 3 (Silabs Next Generation) Platform would be the one targeting AI on edge as they mention x100 AI acceleration?

I also understand that Silabs Series 3 is meant to unifying their multi-radio IoT platform to allow common code base to be used across different wireless protocols.

I have not read anything specific about any new plans for Zigbee with Series 3, though their announcements so far specifically mention multi-protocol, Bluetooth LE, Wi-Fi, Wi-SUN, 15.4, and proprietary protocols, as well as Matter of course.

With that including WiFi it sounds to me like they are also planning to compete with Espressif and ESP32 with that upcoming Silicon Labs Series 3.

Anyway, Silabs Series 3 is also about lower prices and securing supply-chain as switching to 22nm manufacturing will enable them to have parts manufactured in almost any chip-factory in the world so it makes them less depent on the few fabs they are using for Silabs Series 2. So guess that Series 3 might soon cost less than Series 2 as they can probably make higher volumes for less.
